controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -66,8 +69,9 @@ type StorageClassClaimReconciler struct {
 }
 
 const (
-	StorageClassClaimFinalizer  = "storageclassclaim.ocs.openshift.io"
-	StorageClassClaimAnnotation = "ocs.openshift.io.storagesclassclaim"
+	StorageClassClaimFinalizer           = "storageclassclaim.ocs.openshift.io"
+	StorageClassClaimAnnotation          = "ocs.openshift.io.storagesclassclaim"
+	StorageClassClaimNamespaceAnnotation = "ocs.openshift.io.storagesclassclaimnamespacename"
```

Comment:
A claim is a namespaced resource so an annotation that only specifies the name `StorageClassClaimAnnotation` is meaningless. My suggestion is not to have a second annotation for the namespace but change the value of the claim annotation to be `<namespace>/<name>` (which I believe is a common way to represent a namespace name in a string format)

This change will be beneficial for all of the resources that are annotated with this annotation

Reply:
I chosed not to do it because of the length limitation we can it. If we follow this one we will hit name length limitation if the claim name and the namespace name are too long?

Reply:
okay, annotation doesn't have a length limit but labels do. updating as per suggestion but I haven't seen this format anywhere.

Reply:
Yes, I was just going to mention that annotation does not have length limitation